File: Practicas-121/Practica_2/circulo.py
class Punto:
    def __init__(self, x, y):
        self.x, self.y = x, y

    def __str__(self):
        return f"({self.x}, {self.y})"

class Circulo:
    def __init__(self, x, y, r):
        self.centro = Punto(x, y)
        self.radio = r

    def __str__(self):
        return f"Círculo en {self.centro} con radio {self.radio}"

    def dibujar(self):
        print(f"Dibujando {self}")

circulo = Circulo(5, 10, 3.5)
print(circulo)
circulo.dibujar()

File: Practicas-121/Practica_2/circulo.java
package Practica_2;

public class Circulo {
    static class Punto {
        int x, y;
        Punto(int x, int y) { this.x = x; this.y = y; }
        public String toString() { return "(" + x + "," + y + ")"; }
    }

    Punto centro;
    float radio;

    Circulo(int x, int y, float r) {
        centro = new Punto(x, y);
        radio = r;
    }

    public String toString() { return "Círculo en " + centro + " con r=" + radio; }

    void dibujar() { System.out.println("Dibujando " + this); }

    public static void main(String[] args) {
        Circulo c = new Circulo(5, 10, 3.5f);
        System.out.println(c);
        c.dibujar();
    }
}
